src/test/java/hw3/ex2/HomeAndElementsPageVoidTest.java

```diff
@@ -0,0 +1,55 @@
+package hw3.ex2;
+
+import hw3.AbstractTestBase;
+import hw3.ex1.HomePageVoid;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class HomeAndElementsPageVoidTest extends AbstractTestBase {
+
+    @Test
+    public void logInGoToElementsTest() {
+        ElementsPageVoid elementsVoidPage = PageFactory.initElements(driver, ElementsPageVoid.class);
+        HomePageVoid homePageVoid = PageFactory.initElements(driver, HomePageVoid.class);
+        driver.get(credentialsHelper.getProperty("url"));
+        assertEquals(driver.getTitle(), "Home Page");
+
+        //logInAndCheckUserName
+        homePageVoid.openLogInForm();
+        homePageVoid.enterUserName(credentialsHelper.getProperty("login"));
+        homePageVoid.enterPassword(credentialsHelper.getProperty("password"));
+        homePageVoid.clickLoginBtn();
+
+        assertEquals(homePageVoid.getLoggedInUserName(), "ROMAN IOVLEV");
+
+        //getToElementsPage
+        homePageVoid.clickOnTheServiceTab();
+        homePageVoid.clickOnTheDifferentElements();
+
+        //Select checkboxes Water, Wind
+        elementsVoidPage.clickWaterCheckBox();
+        elementsVoidPage.clickWindCheckBox();
+
+        //Select radio selen
+        elementsVoidPage.clickSelenRadio();
+
+        //Select in dropdown Yellow
+        elementsVoidPage.clickColorsDropDown();
+        elementsVoidPage.clickYellow();
+
```

Comment:
It will be better send names of the checkbox, radiobutton and select as parameter to the page method